rmw_cyclonedds_cpp/src/serdata.cpp

```diff
@@ -574,6 +582,49 @@ uint32_t sertype_rmw_hash(const struct ddsi_sertype * tpcmn)
   return h1 ^ h2;
 }
 
+size_t sertype_get_serialized_size(const struct ddsi_sertype * d, const void * sample)
+{
+  const struct sertype_rmw * type = static_cast<const struct sertype_rmw *>(d);
+  try {
+    // ROS 2 doesn't do keys, so its all data (?)
```

Comment:
I would think we do not have to worry about keys as well.

Reply:
Correct, I will make the comment more obvious